projects/go-tester/src/app/app.component.html

```diff
@@ -18,7 +18,7 @@ <h1 class="go-header-2">
 
         <div class="go-column go-column--100">
           <h4>Search</h4>
-          <div style="height: 4rem; background: white; display: flex; align-items: center; padding: 0 1rem;">
+          <div>
```

Comment:
But, where did the styles go?

Reply:
@AlexOverbeck we don't need them, this is just the tester.